wfe/wfe.go

```diff
@@ -1392,6 +1409,100 @@ func (wfe *WebFrontEndImpl) setCORSHeaders(response http.ResponseWriter, request
 	response.Header().Set("Access-Control-Max-Age", "86400")
 }
 
+type rolloverRequest struct {
```

Comment:
Local style has been to define these structs right before the Unmarshal call that uses them, so long as they aren't used in multiple places (as Authorization or Challenge are).